python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -281,6 +282,42 @@ def run_individual(self, number, repeat=1, min_repeat_ms=0):
         ret = self._run_individual(number, repeat, min_repeat_ms)
         return ret.strip(",").split(",") if ret else []
 
+    def run_individual_node(self, index, number, repeat=1, min_repeat_ms=0):
+        """Benchmark a single node in the serialized graph.
+
+        Parameters
+        ----------
+        index : int
+            The index of the node, see `self.debug_datum.get_graph_nodes`
+
+        number: int
+            The number of times to run the node to get a benchmark result.
+
+        repeat: int
+            The number of times to benchmark the nodes.
```

Comment:
Can you use the same language as the `time_evaluator` docs here. Or just point to them.

Reply:
I'll probably just use time_evaluator, so will change later.

Reply:
Done